bundle-workflow/src/build_workflow/build_recorder.py

```diff
@@ -68,11 +68,8 @@ def __init__(self, target):
             self.data["build"]["name"] = target.name
             self.data["build"]["version"] = target.opensearch_version
             self.data["build"]["architecture"] = target.arch
-            self.data["build"]["snapshot"] = str(target.snapshot).lower()
```

Comment:
This is in none of our manifests, checked in the latest build. Not sure what's going on, I would expect the field to be written into them, but I didn't include it in the schema and we can put it back if we need it another time.